e2e/kind.yaml

```diff
@@ -0,0 +1,17 @@
+kind: Cluster
+apiVersion: kind.sigs.k8s.io/v1alpha3
```

Comment:
I'm getting an error trying to start kind for e2e:

```
± make start-kind
kind create cluster \
	 --name secret-manager \
	 --config kind.yaml \
	 --retain \
	 --image "kindest/node:v1.19.0"
ERROR: failed to create cluster: unknown apiVersion: kind.sigs.k8s.io/v1alpha3
make: *** [start-kind] Error 1
```

Reply:
It works if I update `apiVersion` in `e2e/kind.yaml` to:

```yaml
apiVersion: kind.x-k8s.io/v1alpha4
```

which is what they use in [kind-example.config.yaml](https://raw.githubusercontent.com/kubernetes-sigs/kind/master/site/content/docs/user/kind-example-config.yaml)

Reply:
Opened https://github.com/itscontained/secret-manager/pull/57 to fix this.